libs/sdk-ui/src/base/measureFormats/tests/fillMissingFormats.test.ts

```diff
@@ -0,0 +1,21 @@
+// (C) 2007-2022 GoodData Corporation
+import { IInsight, insightMeasures, idMatchMeasure } from "@gooddata/sdk-model";
+import { fillMissingFormats } from "../fillMissingFormats";
+import { insightWithMultipleMeasureBucketsAndFormats } from "../../../../__mocks__/fixtures";
+
+describe("fillMissingFormats", () => {
+    function getMeasureFormat(insight: IInsight, localIdentifier: string): string | undefined {
+        const measures = insightMeasures(insight);
+        const matchingMeasure = measures.find(idMatchMeasure(localIdentifier));
+
+        return matchingMeasure === undefined ? undefined : matchingMeasure.measure.format;
```

Comment:
Nitpick: there is a `measureFormat` getter in sdk-model, please us that instead of "looking" inside of the object structure.